bots/lemmy_sidebar_updater/__init__.py

```diff
@@ -0,0 +1,612 @@
+            team_subs = self.nba_team_subs
+            current_week = None
+        elif self.sport == "NFL":
+            if self.settings.get("NFL", {}).get("TEAM", "") == "":
+                self.log.critical("No team selected! Set NFL > TEAM in Bot Config.")
+                self.bot.STOP = True
+                return
+            nfl = mynflapi.APISession(self.get_nfl_token())
+            current_week = nfl.weekByDate(datetime.now().strftime("%Y-%m-%d"))
+            all_teams = nfl.teams(
+                current_week.get("season", datetime.now().strftime("%Y"))
+            ).get("teams", [])
+            my_team = next(
+                (
+                    x
+                    for x in all_teams
+                    if x["abbreviation"] == self.settings["NFL"]["TEAM"].split("|")[1]
+                ),
+                None,
+            )
+            if self.settings.get("Lemmy", {}).get("STANDINGS_ENABLED"):
+                standings = (
+                    nfl.standings(
+                        season=current_week["season"],
+                        seasonType="REG",
+                        week=current_week["week"]
+                        if current_week["seasonType"] == "REG"
+                        else 18
+                        if current_week["seasonType"] == "POST"
+                        else 1,
+                    )
+                    .get("weeks", [{}])[0]
+                    .get("standings", [])
+                )
+            else:
+                standings = None
+            team_subs = self.nfl_team_subs
+        elif self.sport == "NHL":
+            if self.settings.get("NHL", {}).get("TEAM", "") == "":
+                self.log.critical("No team selected! Set NHL > TEAM in Bot Config.")
+                self.bot.STOP = True
+                return
+            nhl = pynhlapi.API()
+            all_teams = nhl.teams()
+            my_team = next(
+                (
+                    x
+                    for x in all_teams
+                    if x["id"] == int(self.settings["NHL"]["TEAM"].split("|")[1])
+                ),
+                None,
+            )
+            if self.settings.get("Lemmy", {}).get("STANDINGS_ENABLED"):
+                standings = nhl.standings()
+            else:
+                standings = None
+            team_subs = self.nhl_team_subs
+            current_week = None
+
+        self.log.debug(f"{self.sport=}")
+        self.log.debug(f"{my_team=}")
+        self.log.debug(f"{all_teams=}")
+        self.log.debug(f"{standings=}")
+        self.log.debug(f"{team_subs=}")
+        self.log.debug(f"{current_week=}")
+
+        if self.settings.get("Lemmy", {}).get("STANDINGS_ENABLED"):
+            self.log.debug("Updating Lemmy...")
+            self.update_lemmy_standings(
+                my_team,
+                standings,
+                team_subs,
+                all_teams,
+                current_week,
+            )
+
+    mlb_team_subs = {
```

Comment:
Maybe we can alter my URL config to move the json file(s) to a directory under bots where the urls can be shared across all the bots.